za_id_number/za_id_number.py

```diff
@@ -85,14 +66,14 @@ def get_gender(self) -> str:
             else:
                 return Gender.MALE.value
 
-        except Exception as e:
+        except ValueError as e:
```

Comment:
I believe only ValueErrors can be thrown in this code, so I have preferred to catch the more specific Exception.